src/content/admin/cloud-credentials/aws.mdx

```diff
@@ -182,6 +182,9 @@ This grants the IAM Role the necessary permissions to interact with S3.
 
 Now that you've created the IAM role, the final step is to configure the AWS credentials in Okteto.
 
+Cloud Credentials are mainly managed via the [Admin dashboard](admin/dashboard.mdx#cloud-credentials), but can also be managed via one of the CRDs installed by Okteto.
+Follow the steps below to add them via the Admin dashboard, or [follow this guide to add them as a CRD](self-hosted/manage/crds.mdx#cloud-credentials). Please note that if you add credentials using CRDs they will be displayed in the UI, but they can't be modified from the UI.
+
```

Comment:
@pchico83 Do you think it makes sense to add this CRDs call out to the cloud provider guides like this?

Reply:
For consistency, I would follow the same approach as [registry credentials](https://www.okteto.com/docs/1.27/admin/registry-credentials/). If we add the callout here, I would add it to the Registry Credentials doc, too.
Also, more than a call out, I would say the user has 2 options:
- Create it from the UI for a more interactive experience
- Create it using CRDs and GitOps to have more control on changes and show the info "read-only" in the UI
I would even add the CRDs explanation here, and keep the CRDs doc just as a list of our CRDs. Otherwise, that doc will grow a lot as we incorporate more things managed by CRDs

Reply:
Updated this to follow how we do it in registry credentials. I wanted to do this without replicating the CRD guide in multiple places. Think this works ok?